testsuite/config/cucumber.yml

```diff
@@ -32,6 +32,7 @@ retail: --tags @scope_retail
 configuration_channels: --tags @scope_configuration_channels
 content_staging: --tags @scope_content_staging
 proxy: --tags @scope_proxy
+containerized_proxy: --tags @scope_containerized_proxy
```

Comment:
leaving the reminder to not forget to update this for the CI pipelines :stuck_out_tongue:

Reply:
https://github.com/SUSE/susemanager-ci/pull/516